CHANGELOG.md

```
@@ -65,6 +65,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added BuyNow option for Auction [(#533)](https://github.com/andromedaprotocol/andromeda-core/pull/533)
 - Include ADOBase Version in Schema [(#574)](https://github.com/andromedaprotocol/andromeda-core/pull/574)
 - Added multi-hop support for IBC [(#604)](https://github.com/andromedaprotocol/andromeda-core/pull/604)
+- Crowdfund, added additional state: https://github.com/andromedaprotocol/andromeda-core/pull/715
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)